README.md

```diff
@@ -439,6 +473,25 @@ If your certificate or private key have different names, you can set the environ
 
 For letsencrypt certs, we'll autodetect that and add the needed Identrust X3 CA Cert automatically. In case your letsencrypt cert is already the chained certificate, you can set the `CERT_IS_CHAIN` environment variable to `true`, e.g. `CERT_IS_CHAIN=true`. This option also works together with a custom `CERTNAME`.
 
+~~### multiarch~~
+
+~~_[Is this section still true? Relevant? Do the currently-listed Tags make this section obsolete?]_~~
+
+~~All available containers now support multiarch with `amd64`, `armhf`, and `arm64` builds included.
+`armhf` for now uses mongodb 3.4, I do not see much of a path forward for `armhf` due
+to the lack of mongodb support for 32 bit arm, but I will
+support it as long as feasibly possible, for now that date seems to be expiration of support for ubuntu 18.04.~~
+
+~~## Multi-process container~~
+
+~~_[Is this section still true? Relevant? Does the `docker-compose`section supercede this?]_~~
```

Comment:
Multi-process is the default for running docker without compose. There is no real need for the documentation on its behavior but outside of the included docker-compose.yml this behavior is still correct.

Reply:
Good point. I have moved it to the Side Projects document